project/Merging.scala

```diff
@@ -8,6 +8,7 @@ object Merging {
     case PathList("org", "apache", xs @ _*) => MergeStrategy.last
     case PathList("com", "typesafe", xs @ _*) => MergeStrategy.last
     case PathList("com", "google", "auto", "value", xs @ _*) => MergeStrategy.last
+    case PathList("io", "sundr", xs @ _*) => MergeStrategy.last
```

Comment:
the google2 upgrade introduced an assembly conflict between classes in sundr-core and sundr-codegen. Excluding either caused problems, so I'm deduping here during assembly instead.